test/test_metric/test_metrics.py

```diff
@@ -451,6 +520,10 @@ def test_classification_multiclass(self):
             # Skip functions not applicable for multiclass classification.
             if metric in [
                 "roc_auc",
+                "roc_auc_ovo_macro_macro",
```

Comment:
I'm surprised that you add these here. In my opinion, this should be working now because you add the strategies to deal with multiclass classification in roc_auc.